### PR TITLE
[Build] GH actions - coverage, devModel, skip CI on release

### PR DESCRIPTION
**Pull request by mproch**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -16,6 +16,6 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.5.1")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
 
-addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.7")
+addSbtPlugin("net.ruippeixotog"  % "sbt-coveralls"   % "1.3.0")  // fork with scoverage/sbt-coveralls#128 merged in
```
Comment (lciolecki) [inline project/plugins.sbt:19]: Additional spaces? :)

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -61,8 +64,12 @@ jobs:
...
       - name: Backend tests
         shell: bash
-        #TODO: coverage?
-        run: ./ciRunSbt.sh test
+        run: ./ciRunSbt.sh coverage test coverageReport
+      - name: Upload Coverage data to Coveralls
+        shell: bash
+        run: ./ciRunSbt.sh coverageAggregate coveralls
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (lciolecki) [inline .github/workflows/pr.yml:72]: Are you sure that should be `GITHUB_TOKEN`?

**Review by lciolecki — APPROVED** [review]

File: project/plugins.sbt (hunk 1 above)
Comment (mproch) [reply project/plugins.sbt]: :+1: was just copying :)

File: .github/workflows/pr.yml (hunk 2 above)
Comment (mproch) [reply .github/workflows/pr.yml:72]: I just followed: https://github.com/marketplace/actions/coveralls-github-action#inputs
It seems to be working

**Pull request merged**